src/main/java/GuiApp/ConvertedNumberLabel.java

```diff
@@ -10,11 +10,18 @@ public ConvertedNumberLabel(Label label) {
         this.label = label;
     }
 
+    @Override
     public void setText(String text) {
         label.setText(text);
     }
 
+    @Override
     public Label actualLabel() {
         return label;
     }
- }
+
+    @Override
+    public String getText() {
```

Comment:
I don't think you need to add the method `getText` to the interface if you don't need to implement it in the classes that implement the interface. Then you don't need to return null here.